maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -10,7 +10,6 @@
 import java.util.Comparator;
 import java.util.List;
 
-import org.apache.commons.collections.CollectionUtils;
```

Comment:
Since you are not using commons collections anymore, remove it as a dependency in the pom.